src/api.jsx

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
```

Comment:
Nice work exporting the axios calls to a separate file. We can actually make life even easier for ourselves by using axios baseURL. Take a look at the bottom of this page and I'll leave it up to you if you wanna implement it. 🙂 
https://notes.northcoders.com/courses/js-front-end/axios